### PR TITLE
ACS-4461: Documentation updates

### PR DESCRIPTION
**Pull request by mstrankowski**
Adjust default root ca validity to 365 days since we have control over test environments validity.

Remove mentions of 1024 as a possible key size from the Readme files. Add run_ca, run_encryption and run_additional descriptions to Readme files.

I don't think that a docker-compose/docker image is possible here though, especially with run_additional script

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,14 +117,49 @@ Both command line scripts and Docker Image resources can be parametrised by usin
...
+Scripts have been updated to handle multiple Service Alternative Names. To provide multiple of them seperate them with `,`, example: `-servername localhost,additionalSAN`. For Windows variant the value needs to be enclosed in double quotes.
 
 ## Bash Shell Script Standalone (Linux, Mac OS X)
 
-When working on a *Linux* machine, shell script `ssl-tool/run.sh` can be used directly from command line. It's required to have `OpenSSL` and `keytool` programs available in the environment.
+When working on a *Linux* machine, shell scripts can be used directly from command line. It's required to have `OpenSSL` and `keytool` programs available in the environment.
```
Comment (aborroy) [inline README.md:160]: What version of OpenSSL and keytool?

**Review by aborroy — CHANGES_REQUESTED** [review]
LGTM.
Minor comments and the requirement to test adding only "Root CA certificate" to truststores in Repository - Search communication.

File: README.md [hunk 2]
```diff
@@ -6,26 +6,33 @@ This project is not officially supported by Alfresco, as it provides just a samp
 
 As HTTPs invocations happen between different Alfresco services, following relationships must be satisfied:
 
-* Repository is client of SOLR
+* Repository is client of SOLR and Transform Services
```
[the remark this answers is not in the document]
Comment (mstrankowski) [reply README.md:9]: From what I was told during x-in-the-box meeting with Jamal, ActiveMQ is a separate matter, it seems to have its own mechanisms.

File: README.md (hunk 1 above)
Comment (mstrankowski) [reply README.md:160]: Would probably be the best to use latest version, I can add that mention here.

**Review by aborroy — APPROVED** [review]
LGTM

**Review by pzhyland — APPROVED** [review]

**Pull request merged**